### PR TITLE
qa: misc fixes to run for cephfs qa test

### PR DESCRIPTION
**Pull request by lxbsz**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by lxbsz** [comment]
The integration test passed: http://pulpito.ceph.com/xiubli-2020-04-24_08:20:25-fs:basic_functional-wip-lxb-fs-2020-04-24-1447-distro-basic-smithi/

**Comment by lxbsz** [comment]
The integration test passed: http://pulpito.ceph.com/xiubli-2020-04-24_08:20:25-fs:basic_functional-wip-lxb-fs-2020-04-24-1447-distro-basic-smithi/

File: qa/tasks/cephfs/fuse_mount.py [hunk 1]
```diff
@@ -240,15 +240,22 @@ def wait_until_mounted(self):
...
-            stderr = StringIO()
-            self.client_remote.run(args=['sudo', 'chmod', '1777', self.mountpoint], timeout=(15*60), cwd=self.test_dir, stderr=stderr)
-        except run.CommandFailedError:
-            stderr = stderr.getvalue()
-            if "Read-only file system".lower() in stderr.lower():
-                pass
-            else:
-                raise
+        retry = 0
+        while retry < 10:
```
Comment (tchaikov) [inline qa/tasks/cephfs/fuse_mount.py:244]:
```py
for retry in range(10):
  # ...
```

Comment (lxbsz) [reply qa/tasks/cephfs/fuse_mount.py]: Done. Thanks.

**Comment by lxbsz** [comment]
The doc check passed now after @tchaikov 's https://github.com/ceph/ceph/pull/34751 fixing.

**Pull request merged**